CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ listed in the changelog.
 
 - Gradle build dir is now configurable ([#703](https://github.com/opendevstack/ods-pipeline/pull/703))
 
+- ods.yaml branch trigger patterns must be lowercase ([#713](https://github.com/opendevstack/ods-pipeline/issues/713))
```

Comment:
I think this is an outdated changelog. If you rebase on master you'll see that there is a 0.13.2 release and the line you added should actually go above that, into the "Unreleased" section. I would suggest under "Fixed" - or "Changed"?